src/test/app/common/payment-plan/paymentPlan.ts

```diff
@@ -4,153 +4,77 @@ import { Frequency } from 'common/frequency/frequency'
 import { expect } from 'chai'
 import * as moment from 'moment'
 
-const TOTAL_AMOUNT = 1000
+const TOTAL_AMOUNT = 20
 const TOTAL_AMOUNT_2 = 1643.20
```

Comment:
minor: i think more meaningful variable names would be better - e.g. TOTAL_AMOUNT_1000, TOTAL_AMOUNT_20, TOTAL_AMOUNT_1643_20